WordPress/src/main/java/org/wordpress/android/util/WPLegacyMigrationUtils.java

```diff
@@ -77,4 +93,28 @@ private static String getAccessTokenFromTableThenDelete(Context context, String
         }
         return token;
     }
+
+    private static List<SiteModel> getSelfHostedSitesFromDeprecatedDB(Context context) {
+        List<SiteModel> siteList = new ArrayList<>();
+        try {
+            SQLiteDatabase db = context.getApplicationContext().openOrCreateDatabase(DEPRECATED_DATABASE_NAME, 0, null);
+            String[] fields = new String[]{"username", "password", "url"};
+            String byString = "dotcomFlag=0";
+            Cursor c = db.query(DEPRECATED_BLOGS_TABLE, fields, byString, null, null, null, null);
+            int numRows = c.getCount();
+            c.moveToFirst();
+            for (int i = 0; i < numRows; i++) {
+                SiteModel siteModel = new SiteModel();
+                siteModel.setUsername(c.getString(0));
+                siteModel.setPassword(c.getString(1));
+                siteModel.setUrl(c.getString(2));
```

Comment:
We want to set the `xmlRpcUrl` field here (using `setXmlRpcUrl`), not the `url`.